modules/@apostrophecms/asset/lib/webpack/media-to-container-queries-loader.js

```diff
@@ -71,10 +71,10 @@ module.exports = function (source) {
       const newRule = rule.clone();
       newRule.selectors = newRule.selectors.map(selector => {
         if (selector.startsWith('body')) {
-          return selector.replace('body', 'body:not([data-device-preview-mode])');
+          return selector.replace('body', ':not([data-device-preview-mode])');
         }
 
-        return `body:not([data-device-preview-mode]) ${selector}`;
+        return `:not([data-device-preview-mode]) ${selector}`;
```

Comment:
I tried this solution (just having `:not` without `body`) but it did not work. Does it work on Chrome on your end?

Reply:
Indeed not a good idea to remove body, it's fixed